pom.xml

```diff
@@ -2329,19 +2329,26 @@
           </exclusion>
           <exclusion>
             <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
+            <artifactId>jackson-core</artifactId>
           </exclusion>
           <exclusion>
             <groupId>io.netty</groupId>
-            <artifactId>netty-buffer</artifactId>
+            <artifactId>netty-common</artifactId>
           </exclusion>
+        </exclusions>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.arrow</groupId>
+        <artifactId>arrow-memory-netty</artifactId>
```

Comment:
Arrow Java separated the `netty-buffer` dependency into another artifact, to control which allocator is used. The `netty-buffer` was the previous default allocator and the other option is `arrow-memory-unsafe`. The later uses unsafe directly to allocate and release all buffers and does not have the optimizations built in to `netty-buffer`, so I will leave this as-is to maintain performance. It is possible this could be changed and removed in future work.

Reply:
Don't we need any configs except for adding the dependency to indicate which allocator should be used?

Reply:
No, it checks for one on the classpath to use as a default, so it can be set by just including the `netty-buffer` dependency. If there are multiple allocators on the classpath it can be selected by a system property.